ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/contentassist/XMLCompletionProposal.java

```diff
@@ -329,17 +329,12 @@ private void modifyModel(IDocument document) {
 	 */
 	private void findExtensionVirtualPointValue(IPluginBase base) {
 
-		IDocumentRange range = null;
 		PDESourcePage page = fProcessor.getSourcePage();
 		// Ensure page is defined
 		if (page == null) {
 			return;
 		}
-		// When we inserted the extension element and extension point attribute
```

Comment:
Should we preserve this comment?

Reply:
See https://github.com/eclipse-pde/eclipse.pde/pull/98